resolver/resolver.go

```diff
@@ -2879,6 +2902,24 @@ func (r *Resolver) resolveCELValue(ctx *context, env *cel.Env, value *CELValue)
 	if err != nil {
 		return err
 	}
+	var useContextLib bool
+	for _, ref := range checkedExpr.GetReferenceMap() {
+		for _, overloadID := range ref.GetOverloadId() {
+			for _, prefix := range r.ctxOverloadIDPrefixes {
+				if strings.HasPrefix(overloadID, prefix) {
+					useContextLib = true
```

Comment:
nit, we can break this triple for loop here since we don't have to check the rest of the refs